src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -385,8 +386,7 @@ export default {
 		},
 
 		isPrivateReplyable() {
-			return this.isReplyable
```

Comment:
In that case it could be better to not change the behavior, but to refetch messages again
Cause I fixed a button after locking conv., but it still not working after UNlocking